core/lib/network-request.js

```diff
@@ -319,18 +338,18 @@ class NetworkRequest {
 
   /**
    * @param {LH.Crdp.Network.Response} response
-   * @param {number} timestamp
+   * @param {number} ms
```

Comment:
All the other timestamps are doing the s -> ms conversion when set on their respective timing property, so for consistency it seems like it makes sense to do the same in `_onResponse`?